src/components/ContentHeader.vue

```diff
@@ -1,72 +1,62 @@
 <template>
   <div>
-    <div class="usa-overlay"></div>
 <header class="usa-header usa-header--basic">
   <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <div class="usa-logo" id="basic-logo">
-        <em class="usa-logo__text">
-          <a href="javascript:void(0)" title="USGS Vizlab">
-            USGS Vizlab
-          </a>
-        </em>
-      </div>
-      <button class="usa-menu-btn">Menu</button>
-    </div>
     <nav aria-label="Primary navigation" class="usa-nav">
-      <button class="usa-nav__close">
-        <img src="/assets/img/usa-icons/close.svg" role="img" alt="Close" />
-      </button>
       <ul class="usa-nav__primary usa-accordion">
+         <li class="usa-nav__primary-item">
+          <button
+            class="usa-accordion__button usa-nav__link"
```

Comment:
Do we want this to be the 'current' nav upon page load? Right now there's the subtle blue under highlight under 'Visualizations' when you land on the page, which seems a bit odd when you're seeing 'What's new' first:

![image](https://user-images.githubusercontent.com/54007288/136060299-166ac976-0936-447a-a786-985488bb9898.png)

Reply:
Noticing now that that highlight stays even when you click on other links, which is odd. Seems like it should be on whichever was last clicked, *or* if page was just loaded, 'What's new'

Reply:
The `usa-current` class assigned to buttons was driving this. Addressed!